great-idea-website/index.html

```diff
@@ -15,49 +15,71 @@
-  Features
-  About
-  Contact
-
+<nav class= 'Navigation'>
+    <a href="#"Services> </a>
+    <a href="#"Product> </a>
+    <a href="#"Vision> </a>
+    <a href="#"Features></a>
+    <a href="#"About> </a>
+    <a href="#"Contact> </a>
+
+ 
   <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </nav>
+
+  <h1>
   Innovation
   On
   Demand
-
+<div class= "button"
   Get Started
 
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
+  <img src="img/header-img.png" alt="Image of a code snippet."> 
+  </h1>
+ 
+<div class= "top Content">
+  <h2>Features</h2>
   Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
 
 
-  About
+  <h2>About</h2>
   Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
+</div>
+<section></section>
   <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
-  
-  Services
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+</section>
 
+<div>
+  <h2>Services</h2>
+  <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+     </p>
 
-  Product
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
 
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+  <h2>Product</h2>
+  <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+
+  </p>
 
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
+  <h2><Vision/h2>
+  <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
 
-  Copyright Great Idea! 2018
+  </p>
+
+
+</div>
+  
+  <h2>Contact</h2>
+  <address> 123 Way 456 Street
+    Somewhere, USA
+    </address>
+ 
+  1 (888) 888-8888<br>
+
+  <a href= 'mailto:sales@greatidea.io'>sales@greatidea.io</a>
+
+  <footer>
+    <h6>Copyright Great Idea! 2018</h6>
+    </footer>
   
   </body>
 </html>
```

Comment:
Good work with git, you had some road bumps but worked through them and did a great job knowing when to reach out. Be sure to pay attention to detail with your code, as smart as computers are, they aren't intuitive at all so we have to be very literal and exact. You're doing great though, keep it up!